packages/wpcom.js/src/lib/site.media.js

```diff
@@ -158,10 +158,12 @@ Media.prototype.addFiles = function ( query, files, fn ) {
 	}
 
 	const params = {
-		path: '/sites/' + this._sid + '/media/new',
+		path: query.apiPath ?? `/sites/${ this._sid }/media/new`,
 		formData: buildFormData( files ),
 	};
 
+	query.apiPath && delete query.apiPath;
```

Comment:
Is there a good reason to introduce `apiPath` and then delete it instead of just using `path`  and transforming it later before performing the `wpcom.req.post()` request?